content/blog/tyranny-of-nits/index.md

```diff
@@ -0,0 +1,208 @@
+But they're really helpful for people in the thick of the process to consider to!
+Talk about these things clearly and explicitly when you're writing up your design docs or giving feedback.
+It'll help set the stakes, get you taken more seriously, and communicate to the poor author why you're being a pain in the ass.
+
+## Overruling objections: naive consensus is terrible actually
+
+Ultimately though, we want to move proposals through that lifecycle or kill them explicitly.
+
+Instead though, these concerns, both nits and serious architectural complaints, tend to just kill momentum and the thread goes dark.
+Why?
+
+It's because your project **can't make decisions.**
+You have to keep making decisions, day after day, or your momentum will die and everything will slowly rot.
+It doesn't even matter if they're "right": the very fact that a decision was made staves off this decay.
+
+Axiomatically: to make a decision, you have to both be able to say no, and to say yes.
+Open source projects are really good at saying no because they tend to run on naive consensus by default.
+The operating semantics of open source are simple:
+
+1. If everyone says "LGTM", the work gets merged.
+2. If someone reasis a concern, we wait for the author to fix it.
+3. If the author doesn't fix the concern, the PR stays open (or gets closed as stale).
+4. If the author comes back and says "that's not a problem actually", then we get orgnizational undefined behavior.
+
+We even made a [witty, heart-breaking game](
+https://leafwing-studios.itch.io/consensus-together) you can play with your friends/teammates/worst enemies, if you don't believe that naive consensus itself is at fault here.
+
+These rules make open source pretty good at saying yes to uncontroversial things,
+and extremely good at saying no to controversial things, because a conflict-driven stalemate or a timeout due to excessive revisions is still a no.
+
+But that's a *problem*: you have to be able to say yes to controversial decisions too!
+The classical solution here is to have a BDFL, or code owner, who can shut down discussion, merge PRs over objections and close issues as "not planned".
+As long as they're not overworked (never happens), things function smoothly.
+
+But that's not the only patch: delegation, binding votes or [more sophisticated consensus-based decision making processes](https://www.sociocracyforall.org/sociocracy/) all offer another way.
+Just don't ignore this problem!
+
+## Putting it all together
+
+I try to keep my writing positive and light, but for a lot of projects (hi Rust!), I think that this is genuinely their most serious issue.
+If you want [stability without stagnation](https://doc.rust-lang.org/book/appendix-07-nightly-rust.html#stability-without-stagnation), to stop burning people out, to actually make progress on big long term goals: you need to fix this.
```

Comment:
You should probably lead with this, since it seems to be the actual thesis of the post.